cupy/core/__init__.py

```diff
@@ -13,6 +13,7 @@
 from cupy.core._routines_manipulation import broadcast  # NOQA
 from cupy.core._routines_manipulation import broadcast_to  # NOQA
 from cupy.core._routines_manipulation import concatenate_method  # NOQA
+from cupy.core._routines_manipulation import _expand_dims  # NOQA
```

Comment:
I think this alias is not necessary.
You can directly `from cupy.core import _routines_manipulation as _manipulation`.